### DISCUSSION TITLE
Container Data Saving Issue....

### DISCUSSION THREAD

**Discussion by arpit314ad** [Budibase/budibase#1966] [Help]
Hi....

I have setup all date data type in the container but data is not saving. Please look below screen shots...

![Config](https://user-images.githubusercontent.com/86228170/125152471-63147480-e16a-11eb-9790-0f751a016439.JPG)

![output Err](https://user-images.githubusercontent.com/86228170/125152479-70316380-e16a-11eb-98c3-9d705915eac3.JPG)

**Comment by arpit314ad** [comment]
Please advice on the same.

**Reply by mjashanks** [reply]
Hey @arpit314ad, I believe that this is a bug that is logged here: https://github.com/Budibase/budibase/issues/1504

Is there a reason why you would not use the automatic SQL connectors instead of writing a custom query?

Thanks.

**Reply by Manishkparmar80** [reply]
@mjashanks hi , i am not sure about how to use Automatic SQL connectors. Pls share example or doc related to it.

**Reply by mjashanks** [reply]
Hey @Manishkparmar80 - here you go: https://docs.budibase.com/data/data-sources/plus-datasources

No connector for Microsoft Sql Server yet though.

**Reply by Manishkparmar80** [reply]
Hi @mjashanks is MS SQL getting included into Plus Datasource in current month release.
